doc/module-docs/cc_raspberry_pi/example5.yaml

```diff
@@ -1,3 +1,3 @@
 #cloud-config
 rpi:
-  enable_rpi_connect: true
+  enable_usb_gadget: true
```

Comment:
Generally when we drop cloud-config keys (from Ubuntu standpoint), we need to consider the potential upgrade path for cloud images as stable cloud-images receive updates to latest cloud-init version 25.4. In those cases, we still may want old user-data to work, but cloud-init will warn about Deprecation of keys such as the [lock-passwd -> lock_passwd JSON schema example](https://github.com/canonical/cloud-init/blob/25.3/cloudinit/config/schemas/schema-cloud-config-v1.json#L295-L299). It doesn't appear in first blush that this functionality provided via /usr/bin/rpi-usb-gadget is synonymous with the former `/usr/bin/raspi-config", "do_rpi_connect"` call, so maybe this schema example doesn't make sense.